github/Issue.py

```diff
@@ -395,7 +395,8 @@ def get_events(self):
             github.IssueEvent.IssueEvent,
             self._requester,
             self.url + "/events",
-            None
+            None,
+            headers={'Accept': 'application/vnd.github.sailor-v-preview+json'}
```

Comment:
Why do you have to add header here?

Reply:
BTW, all headers now have a designated place:
https://github.com/PyGithub/PyGithub/blob/master/github/Consts.py

Reply:
It was introduced in this ae5cdb285e2d41e55b2f6fac56fd4eff239c6309

Reply:
Issue.get_events() returns a IssueEvent object, we need the header here to ensure we get the lock_reason attribute.